lib/msal-common/test/client/BaseClient.spec.ts

```diff
@@ -88,4 +106,268 @@ describe("BaseClient.ts Class Unit Tests", () => {
             );
         });
     });
+
+    describe("sendPostRequest tests", () => {
+        it("returns a response", async () => {
+            const config =
+                await ClientTestUtils.createTestClientConfiguration();
+            const client = new TestClient(config);
+
+            const thumbprint: RequestThumbprint = THUMBPRINT;
```

Comment:
nit: is this line necessary? You're using a constant directly on line 158. So why make this variable?

Reply:
Same comment for the options variable directly below this

Reply:
It might not be, these tests were simply moved, if we want to make updates I'd suggest we track that as a separate item.